docusaurus.config.js

```diff
@@ -108,6 +108,11 @@ module.exports = {
       successMessage: "Success! ✓",
       errorMessage: "Sorry, there was issue sending your email. Please try again.",
     },
+    promoBanner: {
+      text: "Don't miss the BIGGEST Airflow event of the year: Airflow Summit 2023 - Sept 19-23",
```

Comment:
```suggestion
      text: "Don't miss the biggest Airflow event of the year: Airflow Summit 2023, Sept 19-23",
```
Can I executive override some of this copy haha

Reply:
Totally! haha